Python_Backend/DFGenerator.py

```diff
@@ -278,14 +278,10 @@ def __display_debug_image(landmarks, image):
         
         if landmarks != None:
             for faceLandmarks in landmarks:
-                FacialDFGenerator.__drawingModule.draw_landmarks(image, faceLandmarks, FacialDFGenerator.__faceModule.FACEMESH_CONTOURS, 
+                FacialDFGenerator.__drawingModule.draw_landmarks(image, faceLandmarks, FacialDFGenerator.__faceModule.FACE_CONTOURS, 

```

Comment:
Idk why the Facemesh isn't working on your end, but for now, let's just keep it as this. Honestly, if it works for the machine learning it makes no difference to me.

Reply:
Agree - will change that  - I think we need to give an explicit state for debug though as currently when this method runs the program hangs.

Reply:
Opened #134 to address debug mode